requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy>=1.22.0rc1
 scikit-learn==1.0.2
 scipy==1.8.0
-dataprofiler==0.7.10
\ No newline at end of file
+dataprofiler==0.8.0
```

Comment:
DP version 0.8.0 is required in order to reflect the changes made to graph profiles.